freqtrade/configuration/check_exchange.py

```diff
@@ -27,6 +27,14 @@ def check_exchange(config: Dict[str, Any], check_for_bad: bool = True) -> bool:
     logger.info("Checking exchange...")
 
     exchange = config.get('exchange', {}).get('name').lower()
+    if not exchange:
+        raise OperationalException(
+            f'This command requires a configured exchange. You can use either '
+            f'`--exchange <exchange_name` or use a configuration via `--config`.\n'
```

Comment:
* "You should either use"
* `<exchange_name>`
* "or specify a configuration file via"